packages/date-picker/test/dom/month-calendar.test.js

```diff
@@ -1,20 +1,30 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
 import '../../src/vaadin-month-calendar.js';
 import { resetUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
 import { getDefaultI18n } from '../helpers.js';
 
 describe('vaadin-month-calendar', () => {
-  let monthCalendar;
+  let monthCalendar, clock;
 
   beforeEach(async () => {
     resetUniqueId();
+    clock = sinon.useFakeTimers({
+      now: new Date('2016-02-05'),
```

Comment:
"date-only forms are interpreted as a UTC time", as time is irrelevant here, can you help me understand why this would be a problem? Why isn't this a problem in the `max date` test just below? https://github.com/vaadin/web-components/blob/f50d8c2b3fe7ad242277adbd09e9f088aa1a084e/packages/date-picker/test/dom/month-calendar.test.js#L30

Reply:
In any case, what can I do about it? Explicitly set the time and timezone as well?

Reply:
If I set my location to "San Francisco" for example:
<img width="788" alt="Screenshot 2024-08-27 at 11 48 28" src="https://github.com/user-attachments/assets/2293ed15-12e4-45ed-98b1-44c2e28c3b54">

Then:
<img width="293" alt="Screenshot 2024-08-27 at 11 46 38" src="https://github.com/user-attachments/assets/4686829c-dca6-4e5c-a8e7-62a618088bea">

Reply:
```suggestion
      now: new Date(2016, 1, 5),
```